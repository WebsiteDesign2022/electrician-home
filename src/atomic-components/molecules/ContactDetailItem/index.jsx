import React from "react";

import "./index.scss";

const ContactDetailItem = ({ title, content }) => (
    <div className="feature">
        <img src="" />
        <h5 align="left">{title}</h5>
        <p align="left">{content}</p>
    </div>
);

export default ContactDetailItem;
