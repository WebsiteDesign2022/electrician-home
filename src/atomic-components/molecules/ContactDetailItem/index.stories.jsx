import React from 'react';

import ContactDetailItem from '.';

export default {
    title: 'Electrician-Home/molecules/ContactDetailItem',
    component: ContactDetailItem,
    parameters: {
      // More on Story layout: https://storybook.js.org/docs/react/configure/story-layout
      layout: 'fullscreen',
    },
};

const Template = (args) => <ContactDetailItem {...args} />;

export const Primary = Template.bind({});
// More on args: https://storybook.js.org/docs/react/writing-stories/args
Primary.args = {

};
