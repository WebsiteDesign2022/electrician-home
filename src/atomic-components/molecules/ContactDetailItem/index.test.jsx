import TestRenderer from 'react-test-renderer';

import ContactDetailItem from '.';

describe("ContactDetailItem", () => {
    test("renders ContactDetailItem", () => {
        const contactDetailItem = TestRenderer.create(
            <ContactDetailItem />
        );

        expect(contactDetailItem.toJSON()).toMatchSnapshot();
    });
});
