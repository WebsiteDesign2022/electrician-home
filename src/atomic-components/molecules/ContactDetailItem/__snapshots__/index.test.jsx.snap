// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactDetailItem renders ContactDetailItem given empty 1`] = `
<div
  className="feature"
>
  <img
    src=""
  />
  <h5
    align="left"
  />
  <p
    align="left"
  />
</div>
`;

exports[`ContactDetailItem renders ContactDetailItem given non-empty content 1`] = `
<div
  className="feature"
>
  <img
    src=""
  />
  <h5
    align="left"
  />
  <p
    align="left"
  />
</div>
`;
